README.md

```diff
@@ -1136,6 +1136,10 @@ If you are using node@10 and puppeteer 1.4.0 or less, you may experience message
 
 > Remember to check [Puppeteer Troubleshooting](https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md)
 
+## Change Logs
```

Comment:
Just to maintain the same style, please change this to Changelog

Reply:
Fixed in PR #155